docs/concepts/data-views.asciidoc

```diff
@@ -32,51 +30,68 @@ uploaded a file, or added sample data,
+
+. Click *Create a {data-source}*.
 
-. Click *Create {data-source}*.
+. Give your {data-source} a name.
 
-. Start typing in the *name* field, and {kib} looks for the names of
+. Start typing in the *Index pattern* field, and {kib} looks for the names of
 indices, data streams, and aliases that match your input.
 +
 [role="screenshot"]
 image:management/index-patterns/images/create-data-view.png["Create data view"]
 +
-** To match multiple sources, use a wildcard (*). For example, `filebeat-*` matches
+** To match multiple sources, use a wildcard (*). `filebeat-*` matches
 `filebeat-apache-a`, `filebeat-apache-b`, and so on.
 +
 ** To match multiple single sources, enter their names,
 separated by a comma.  Do not include a space after the comma.
-`filebeat-a,filebeat-b` matches two indices, but does not match `filebeat-c`.
+`filebeat-a,filebeat-b` matches two indices.
 +
 ** To exclude a source, use a minus sign (-), for example, `-test3`.
 
-. If {kib} detects an index with a timestamp, expand the *Timestamp field* menu,
+. Open the *Timestamp field* dropdown,
 and then select the default field for filtering your data by time.
 +
-** If your index doesn’t have time-based data, choose *I don’t want to use the time filter*.
-+
 ** If you don’t set a default time field, you can't use
 global time filters on your dashboards. This is useful if
 you have multiple time fields and want to create dashboards that combine visualizations
-based on different timestamps.
+based on different timestamps.z
++
+** If your index doesn’t have time-based data, choose *I don’t want to use the time filter*.
 
-. To display all indices, click *Show advanced settings*, then select *Allow hidden and system indices*.
+. Click *Show advanced settings* to:
+** Display hidden and system indices.
+** Specify your own {data-source} name. For example, enter your {es} index alias name.
 
-. To specify your own {data-source} name, click *Show advanced settings*, then enter the name in the *Custom {data-source} ID* field. For example, enter your {es} index alias name.
+. [[reload-fields]] Click *Save {data-source} to {kib}*.
++
+You can manage your data view from *Stack Management*.
 
-. [[reload-fields]] To use this data in searches and visualizations that you intend to save, click *Save {data-source} to {kib}*.
+[float]
+==== Create a temporary {data-source}
 
-. To explore your data without creating a {data-source}, click *Use without saving*.
-+
-This allows you to explore your data in *Discover*, *Lens*, and *Maps*
-without making it visible to others in your space. You can save the {data-source} later
-if you create a search or visualization that you want to share.
+Want to explore your data or create a visualization without saving it as a data view?
+Select *Use without saving* in the *Create {data-source}* form in *Discover*
+or *Lens*. With a temporary {data-source}, you can
+do everything you'd do with a saved
+{data-source},
```

Comment:
well, in Discover you can't use "Share", this will convert the the temporary into a regular data view.